server/sonar-web/src/main/js/apps/tutorials/onboarding/TokenStep.js

```diff
@@ -118,26 +118,26 @@ export default class TokenStep extends React.PureComponent {
         {token != null
           ? <form onSubmit={this.handleTokenRevoke}>
               {tokenName}{': '}
-              <span className="monospaced spacer-right">{token}</span>
+              <span className="monospaced spacer-right text-middle">{token}</span>
               {loading
-                ? <i className="spinner" />
-                : <button className="button-clean" onClick={this.handleTokenRevoke}>
+                ? <i className="spinner text-middle" />
+                : <button className="button-clean text-middle" onClick={this.handleTokenRevoke}>
```

Comment:
I'm not sure about those last 3 `.text-middle` for the generated token. The delete button doesn't seem well aligned. To me it looks better without the 3 `.text-middle` classes.